kits/arm/ex_AR_kit.py

```diff
@@ -105,26 +105,26 @@ def quat2rotMat(q):
     continue
 
   # B1 - Return to home position
-  if m.get_button_diff(1) == 3: # "ToOn"
+  if m.get_button_diff(1) == 1: # "ToOn"
```

Comment:
this is technically going to be a backwards incompatible change, I guess -- perhaps worth either renaming the function and deprecating the old one?  Unless this was marked experimental?